web/src/lib/cms/previews/roadmap-sidebar-card.tsx

```diff
@@ -22,9 +22,13 @@ const RoadmapSidebarCardPreview = (props: Props) => {
   const { body, ...data } = JSON.parse(JSON.stringify(props.entry.getIn(["data"])));
   const card: SidebarCardContent = { ...data, contentMd: body };
 
+  const cTAOnClickMap = {
```

Comment:
This c feels like it should be a big C

Reply:
I think it should be little cta, like `ctaOnClickMap`

Reply:
Sounds good to me.